app/controllers/friendships_controller.rb

```diff
@@ -1,21 +1,30 @@
 class FriendshipsController < ApplicationController
-  include ApplicationHelper
-  include FriendshipsHelper
   before_action :authenticate_user!
   def new; end
 
   def create
     friend_id = params[:friend_id]
     @friendship = Friendship.find_or_create_by(user_id: current_user.id, friend_id: friend_id)
+    @friendship = Friendship.find_or_create_by(user_id: friend_id, friend_id: current_user.id)
   end
 
   def update
     user = User.find(params[:user_id])
-    confirm_a_request(user)
+    friend = User.find(current_user.id)
+    @friendship = Friendship.find_by_user_id_and_friend_id(user, friend)
+    @friendship.confirmed = true
+    @friendship.save
+    @friendship = Friendship.find_by_user_id_and_friend_id(friend, user)
+    @friendship.confirmed = true
+    @friendship.save
+    redirect_to users_path
   end
```

Comment:
When a user sends an invitation (in the `#create` action), only one row is created in the friendships table with `confirmed` set to false. And when that invitation is accepted  (in the `#update` action), another row with inverted attributes is created.

Please rectify the code to reflect the above behavior.